keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -707,9 +738,9 @@ Enabling the feature does not change any default behavior.
 Yes, if the ClusterTrustBundle feature gate is disabled, the cluster will
 largely continue to function properly.
 
-Any pods that specify a pemTrustAnchors projected volume source will encounter
-errors as kubelet will no longer process their spec.  Existing pods will stop
-seeing updates to their PEM files, and new pods will be rejected.
+Any pods that specify a ClusterTrustBundle projected volume source will
+encounter errors as kubelet will no longer process their spec.  Existing pods
+will stop seeing updates to their PEM files, and new pods will be rejected.
 
 Any ClusterTrustBundle objects in the cluster will stop being served, but be
 retained in etcd storage.
```

Comment:
`### Rollout, Upgrade and Rollback Planning` and its sub sections need to be filled out.